composer.json

```
@@ -8,29 +8,52 @@
             "name": "Anton Titov (wolfy-j)",
             "email": "wolfy-j@spiralscout.com"
         },
+        {
+            "name": "Pavel Buchnev (butschster)",
+            "email": "pavel.buchnev@spiralscout.com"
+        },
+        {
+            "name": "Aleksei Gagarin (roxblnfk)",
+            "email": "alexey.gagarin@spiralscout.com"
+        },
+        {
+            "name": "Maksim Smakouz (msmakouz)",
+            "email": "maksim.smakouz@spiralscout.com"
+        },
         {
             "name": "RoadRunner Community",
             "homepage": "https://github.com/spiral/roadrunner/graphs/contributors"
         }
     ],
+    "homepage": "https://roadrunner.dev/",
+    "support": {
+        "docs": "https://roadrunner.dev/docs",
+        "issues": "https://github.com/roadrunner-server/roadrunner/issues",
+        "forum": "https://forum.roadrunner.dev/",
+        "chat": "https://discord.gg/V6EK4he"
+    },
+    "funding": [
+        {
+            "type": "github",
+            "url": "https://github.com/sponsors/roadrunner-server"
+        }
+    ],
     "bin": [
         "bin/rr"
     ],
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.1",
         "ext-json": "*",
         "composer/semver": "^3.2",
-        "spiral/roadrunner-worker": ">=2.0.2",
-        "spiral/tokenizer": "^2.13 || ^3.0",
-        "symfony/console": "^4.4|^5.0|^6.0",
-        "symfony/http-client": "^4.4|^5.0|^6.0",
-        "symfony/polyfill-php80": "^1.22",
-        "symfony/yaml": "^5.4 || ^6.0"
+        "spiral/roadrunner-worker": ">=3.0",
```

```suggestion
        "spiral/roadrunner-worker": "^3.0",
```

Is this dependency used?